docs/source/tutorials/dash_hls_example.rst

```diff
@@ -11,3 +11,21 @@
 
 The above packaging command creates five single file MP4 streams, and HLS
 playlists as well as DASH manifests.
+
+* Output DASH + HLS with dash_only and hls_only options
+
+    $ packager \
+      'in=h264_baseline_360p_600.mp4,stream=audio,init_segment=audio/init.mp4,segment_template=audio/$Number$.m4s' \
+      'in=input_text.vtt,stream=text,init_segment=text/init.mp4,segment_template=text/$Number$.m4s',dash_only=true' \
```

Comment:
Oops, I accidentally included an extra ' after "m4s", can you remove it? Same below.